samples/posix/gettimeofday/src/main.c

```diff
@@ -16,15 +16,19 @@ int main(void)
 
 	while (1) {
 		int res = gettimeofday(&tv, NULL);
+		time_t now = time(NULL);
```

Comment:
So, the sample is called "samples/posix/gettimeofday", and initially was intended to showcase gettimeofday (and not less than that, system time auto-init via SNTP). If you add more function calls there, please at least add comment like "Also exercise a few other functions at the same time."

Reply:
Updated the example description to note that localtime is printed as well.